package.json

```diff
@@ -21,7 +21,7 @@
 		"e2e": "ng e2e",
 		"fix-packages": "node scripts/fix-packages",
 		"fix-package-json": "node scripts/fix-package-json",
-		"fix-all": "npm run fix-packages && npm run fix-package-json",
+		"fix-all": "npm run fix-packages && npm run fix-package-json && git add -A",
```

Comment:
git add should not be part of fix script  / commit hook but rather be used explicitly.

Reply:
why not?
'fix-packages' script fix your 'imports', but changes isn't includes on the commit.
'pre-commit'  is finished with your '../pakcages/' old mistakes.